apps/ident-service/src/main/java/no/nav/registre/testnorge/identservice/testdata/config/RestConsumerConfig.java

```diff
@@ -0,0 +1,38 @@
+package no.nav.registre.testnorge.identservice.testdata.config;
+
+import org.apache.http.client.config.RequestConfig;
+import org.apache.http.conn.ssl.DefaultHostnameVerifier;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.impl.conn.SystemDefaultRoutePlanner;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.web.client.RestTemplate;
+
+import java.net.ProxySelector;
+
+
+@Configuration
+public class RestConsumerConfig {
+
+    private static final int TIMEOUT = 45_000;
+
+    @Bean
+    public RestTemplate restTemplate(RestTemplateBuilder restTemplateBuilder) {
```

Comment:
Ikke bruk RestTemplate da den er:
"NOTE: As of 5.0 this class is in maintenance mode, with only minor requests for changes and bugs to be accepted going forward. Please, consider using the org.springframework.web.reactive.client.WebClient which has a more modern API and supports sync, async, and streaming scenarios."

https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/client/RestTemplate.html

Bruk webflux: https://www.baeldung.com/spring-webflux

Reply:
Enda jeg klarer ikke helt så hvor du bruker restTemplate? Er denne brukt?

Reply:
Nei, den er copy paste fra TPSF, tror ikke den er i bruk så skal fjerne 👍